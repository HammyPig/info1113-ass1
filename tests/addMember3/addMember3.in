MEMBER 100000
ADD MEMBER mr meeseeks
add member mr meeseeks
aDD mEMBer mr meeseeks
MEMBER 100000
MEMBER 100001
MEMBER 100002