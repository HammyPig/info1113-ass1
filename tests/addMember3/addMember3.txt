can add members with the same name