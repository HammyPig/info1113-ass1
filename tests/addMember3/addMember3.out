user: No members in system.

user: Success.

user: Success.

user: Success.

user: 100000: mr meeseeks

user: 100001: mr meeseeks

user: 100002: mr meeseeks

user: Ending Library process.
